intro/classes/index.html

```diff
@@ -129,37 +122,31 @@ <h2><a id="simple-class" class="js"></a>Creating a Simple Class</h1>
+            and some inventory to something called <code>self</code>. The <code>self</code> parameter represents the <i>class instance</i> (or in other words, it's 
+            <i>referring to itself</i>), so you're really just assigning those bits of data to the instance of the class. This serves a quite useful purpose. 
+            You see, if I didn't prefix all of those with <code>self</code>, then when the constructor finished running the values would get garbage collected 
+            away and would not exist anymore. That's not what we want at all. So, we assign them to be instance attributes as above and those values stay in 
+            scope for as long as the instance itself exists. Another benefit is that we can change them for individual instances and the changes 
+            only apply to that single instance and no others.
+            Here's an example:
+
+            <iframe width='800px' height='200px' class="python-embed" src='https://godbolt.org/e#z:OYLghAFBqd5QCxAYwPYBMCmBRdBLAF1QCcAaPECAMzwBtMA7AQwFtMQByARg9KtQYEAysib0QXACx8BBAKoBnTAAUAHpwAMvAFYTStJg1AAHAJ4EEA0kvrICeAZUboAwqloBXFgwnTnAGTwGTAA5LwAjTGIQACYAZlJjVAVCBwY3T29fROTUgUDgsJZI6PjrTFt7ASECJmICDK8fKXLKtJq6ggLQiKjYhIVa%2BsaslsHO7qKS/oBKa1QPYmR2DmQDBQUAahDlFxAAUg0AQU3TzawqTYB9K6DCG4gbKhmD47P3zaeAOgRMMQtNvs4gARTZcDQaQ4nD6nb5MAi1ZAAa0BIM2AFYoTDYRUqF8LowlKjQZC3tjvkEAG6MIjEUzEwHogBCYA4CgA7iR0KzSJtWQoEHgKtyOPt0cCsdiklUGFdMFQqJg7FsgaCxSy2QRiIxgBYeXy2ey8FQCMENqyxRLjlCGMZkFwGTsXBAZja7TFHbsXVC3fb8fLCZgGZjjrbkDEvvDESjVWDSUcocZiEECNR9jEYvsAOxMsNcH5/WgA2PZiUZ13HJMptMZ7O5u35qNMZEM0vpmIVo5VwQ1zM5vP%2BxUMIklrNljuJ5M9qjtusDqk0kj00fjzuT6sz2v990F/4IVtj9ud7upzd9%2BvhyMI5sxtFt8vr6ez7eXgnDoMro%2BP0/Pi8RheCEuB6rj6lZTj%2BW4Xvm0ppHKCpKgQKp3oe5YcHMtCcOivA%2BBwWikKgnDKOYlgMJ8CxLB%2B8Q8KQBCaGhcxIiA6IaPonCSNhdH4ZwvAKCALG0bhaGkHAsBIJgqhKh4tLkJQdTAAoyiGBUQiWOyOHUWgLDGHQ8JpIpwS0CpqBqZwGmoFpdB9GYFgCHE4KkJp2n0MQADyUlGSZgmkOJSpHMQ8ncd5EnIDU%2BA4bw/CCCIYjsC0kXyEoaicboXD6IYJjEVYtB4OEvGQHMqDGDKvEcDx5HLHogxhfpymqepvBapgKzUeyxBMMYplCRhHBYaQOF4QRHDYMFUkkJsRE2aRcRfOCmwQBNJG8rghBjemdkzA1dEzAxTEsd17F9Zxg08XxNFbehnAxBxXnHWdgnbaQ1LECkjiSEAA%3D%3D'></iframe>
+
+            We can see that all of the attributes we prefixed with <code>self</code> printed just fine, but as soon as we tried to print the value that wasn't prefixed,
+            we got an <code>AttributeError</code> which means that the attribute we tried to access does not exist.
         </p>
 
         <p>
-            A general rule of thumb for whether something should be an attribute
-            or not is this:
+            A general rule of thumb for whether something should be an attribute or not is this:
             <blockquote>
-                Make it an attribute if you need it later and ONLY if you need
-                it later
+                Make it an attribute if you need it later and ONLY if you need it later
             </blockquote>
         </p>
 
         <p>
```

Comment:
I love this section, but I see that it's a subsection under "Creating a Simple Class". Should we move it somewhere else?

Reply:
I think it's fine here for now. I can't think of any better place in the article to put it at this point in time

Reply:
ah, alright 👍🏽